modules/distribution/src/main/scripts/impl/init.sh

```diff
@@ -32,6 +32,11 @@ if [[ -z $HADOOP_PREFIX ]]; then
   exit 1
 fi
 
+if [ ! -d "$HADOOP_PREFIX" ]; then
```

Comment:
I initially had it there but `fluo mini` should not require that `$HADOOP_PREFIX` is set and this only needed for the `init` & `yarn` commands